### PR TITLE
Feature/add users card bl

### PR DESCRIPTION
**Pull request by YarynaBas1234**

### CODE REVIEW

File: src/containers/Home/Card.tsx [hunk 1]
```diff
@@ -1,23 +1,53 @@
...
-   }
+  margin: 16px 16px 0 0;
 `;
 
-export const Card: React.FC = () => {
+const UserName = styled(H4)`
+  text-align: center;
+`;
+
+const Data = styled(H5)`
```
Comment (illia-nerdysoft) [inline src/containers/Home/Card.tsx]: bad naming

File: src/containers/Home/Card.tsx [hunk 2]
```diff
@@ -1,23 +1,53 @@
...
   padding: 16px;
-  
-  :not(:last-child) { 
-    margin-right: 8px;
-   }
+  margin: 16px 16px 0 0;
 `;
 
-export const Card: React.FC = () => {
+const UserName = styled(H4)`
```
Comment (illia-nerdysoft) [inline src/containers/Home/Card.tsx]: bad naming

File: src/containers/Home/Card.tsx [hunk 3]
```diff
@@ -1,23 +1,53 @@
...
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const DataTitle = styled(H5)`
+  margin-right: 8px;
+`;
+
+interface ICard {
+  item: IUser;
```
Comment (illia-nerdysoft) [inline src/containers/Home/Card.tsx]: item - bad naming. use appropriate namings

File: src/containers/Home/Card.tsx [hunk 4]
```diff
@@ -1,23 +1,53 @@
...
+`;
+
+const Data = styled(H5)`
+  margin-top: 16px;
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const DataTitle = styled(H5)`
+  margin-right: 8px;
```
Comment (illia-nerdysoft) [inline src/containers/Home/Card.tsx]: try to avoid right margins

File: src/translations/ua.json [hunk 5]
```diff
@@ -36,6 +36,10 @@
       "SKYPE": "Skype:",
       "FACEBOOK": "Facebook:"
     },
+    "HOME_PAGE": {
+      "EMAIL": "логін: ",
```
Comment (illia-nerdysoft) [inline src/translations/ua.json]: i guess, that here is bad translation

File: src/containers/Home/Home.tsx [hunk 6]
```diff
@@ -1,21 +1,28 @@
...
+  users: IUser[] | null;
+}
+
+export const Home: React.FC<IHome> = (props) => {
+  const { users } = props;
 
   return (
     <HomeWrapper>
-      <Card />
+      {users?.map(item => {
```
Comment (illia-nerdysoft) [inline src/containers/Home/Home.tsx]: move it to the separate constant, use appropriate hook and simplify this

File: src/containers/Home/Card.tsx [hunk 7]
```diff
@@ -1,23 +1,53 @@
...
+  width: 100%;
+  background-color: ${({ theme }) => theme.colors.lightBlue};
   display: flex;
   flex-direction: column;
   padding: 16px;
-  
-  :not(:last-child) { 
-    margin-right: 8px;
-   }
+  margin: 16px 16px 0 0;
```
Comment (illia-nerdysoft) [inline src/containers/Home/Card.tsx]: use left margin instead of right

**Review by illia-nerdysoft — CHANGES_REQUESTED** [review]

File: src/containers/Home/Card.tsx [hunk 8]
```diff
@@ -1,23 +1,53 @@
...
+  text-align: center;
+`;
+
+const UserDataWrapper = styled(H5)`
+  margin-top: 16px;
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const UserData = styled(H5)`
```
Comment (illia-nerdysoft) [inline src/containers/Home/Card.tsx]: Bad naming

File: src/containers/Home/Card.tsx [hunk 9]
```diff
@@ -1,23 +1,53 @@
...
-   }
+  margin: 16px 16px 0 16px;
 `;
 
-export const Card: React.FC = () => {
+const Title = styled(H4)`
+  text-align: center;
+`;
+
+const UserDataWrapper = styled(H5)`
```
Comment (illia-nerdysoft) [inline src/containers/Home/Card.tsx]: Bad naming

File: src/containers/Home/Home.tsx [hunk 10]
```diff
@@ -2,20 +2,23 @@ import React from 'react';
...
+  CreateUsersCard: JSX.Element[] | undefined;
+}
+
+export const Home: React.FC<IHome> = (props) => {
+  const { CreateUsersCard } = props;
 
   return (
     <HomeWrapper>
-      <Card />
+      {CreateUsersCard}
```
Comment (illia-nerdysoft) [inline src/containers/Home/Home.tsx]: Bad naming

File: src/containers/Home/index.tsx [hunk 11]
```diff
@@ -1,18 +1,23 @@
...
 
-  return (
-    <Home />
-  );
-}
+  const CreateUsersCard = React.useMemo(() => {  
+    return users?.map((user) => <Card key={user.id} user={user} />);
+  }, [users]);
+
+  return <Home CreateUsersCard={CreateUsersCard}/>;
```
Comment (illia-nerdysoft) [inline src/containers/Home/index.tsx]: Bad naming

**Review by illia-nerdysoft — CHANGES_REQUESTED** [review]

File: src/containers/Home/Card.tsx [hunk 12]
```diff
@@ -1,23 +1,53 @@
...
+
+export const Card: React.FC<ICard> = (props) => {
+  const { user } = props;
+  const { t } = useTranslation();
 
   return (
-    <CardWrapper>User`s card</CardWrapper>
+    <CardWrapper>
+      <Title>{user.name}</Title>
+      <UsersDataWrapper>
```
Comment (illia-nerdysoft) [inline src/containers/Home/Card.tsx]: move to the new component and rename inner components

File: src/containers/Home/Home.tsx [hunk 13]
```diff
@@ -2,20 +2,23 @@ import React from 'react';
...
-  margin-top: 16px;
-  border-radius: 8px;
+  flex-wrap: wrap;
+  padding: 16px;
+  justify-content: space-around;
 `;
 
-export const Home: React.FC = () => {
+interface IHome {
+  createUsersCard: JSX.Element[] | undefined;
```
Comment (illia-nerdysoft) [inline src/containers/Home/Home.tsx]: you can simplify this

Comment (illia-nerdysoft) [inline src/containers/Home/Home.tsx]: React.ReactElement

Comment (illia-nerdysoft) [inline src/containers/Home/Home.tsx]: use only one element instead of array

Comment (illia-nerdysoft) [inline src/containers/Home/Home.tsx]: rename appropriate to the type

File: src/containers/Home/index.tsx [hunk 14]
```diff
@@ -1,18 +1,23 @@
...
 
   React.useEffect(() => {
     dispatch(handleGetUsersAction);
   }, []);
 
-  return (
-    <Home />
-  );
-}
+  const createUsersCard = React.useMemo(() => {
```
Comment (illia-nerdysoft) [inline src/containers/Home/index.tsx]: read about use memo

Comment (illia-nerdysoft) [inline src/containers/Home/index.tsx]: JSX should not be in the index file. move in inside container

File: src/containers/Home/index.tsx [hunk 15]
```diff
@@ -1,18 +1,23 @@
...
   React.useEffect(() => {
     dispatch(handleGetUsersAction);
   }, []);
 
-  return (
-    <Home />
-  );
-}
+  const createUsersCard = React.useMemo(() => {  
+    return users?.map((user: IUser) => <Card key={user.id} user={user} />);
```
Comment (illia-nerdysoft) [inline src/containers/Home/index.tsx]: return only one element here

**Review by illia-nerdysoft — CHANGES_REQUESTED** [review]

**Review by ArturFFor — APPROVED** [review]

**Pull request merged**